.config/playwright-setup.js

```diff
@@ -107,10 +106,11 @@ function kill_process(process) {
 }
 
 function make_app(demos, port) {
-	return `import gradio as gr
```

Comment:
There is another `import gradio as gr` just below, so deleted this duplicated import.